Covid Lunches/express/express-bootstrap/.gitignore

```diff
@@ -1 +1,2 @@
-config.json
\ No newline at end of file
+config.json
+public/config.js
```

Comment:
Should not be in public

Reply:
Resolved with most recent commit. It runs in config.js now.